cprint/color_test.go

```diff
@@ -0,0 +1,88 @@
+package cprint
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/stretchr/testify/assert"
+)
+
+func captureOutput(f func()) string {
```

Comment:
Sorry about bugging you again - forgot about this. We should probably revert the prior setting of `color.Output` and `defer` setting it back.

Reply:
Updated!